docs/getting-started/guides/how-to-understand-strategies-descriptions.md

```diff
@@ -0,0 +1,111 @@
+# How to Understand Strategies Descriptions
+
+Navigating strategies descriptions at [yearn.watch](https://yearn.watch/) or [vaults.yearn.finance](https://vaults.yearn.finance/) may expose you to terms that you have never seen, this page contains a complete glossary on what each term means when used to describe strategies for yearn vaults.
+
+It complements our DeFi [Glossary](../../resources/defi-glossary) and it's more focused only on the words used by strategy descriptions and shorter explanations.
+
+- Regular links navigate between the glossary.
+- Links marked with ↗ go to external websites.
+
+# Yearn
+
+### yVault / Yearn Vault
+- [Smart contract↗](https://ethereum.org/en/developers/docs/smart-contracts/) that holds user deposits and allocates into different strategies (up to 20 in the current version 2).
+### yvToken / Token
```

Comment:
I wouldn't group this two together. 

A `yvToken` is a `Token` that represents the share of a yVault. 

We use `Token` as a reference to ERC20 smart contracts that represent a certain value, ie DAI

The above is an example, but I think it's important to say that is a reference we are using here. There can be more complex definitions of a token. And a yvToken is just a category of tokens that we created. That's why I wouldn't group them together, to avoid confusion